internal/cli/clients/client.go

```diff
@@ -88,6 +89,46 @@ func NewEpinioClient(ctx context.Context, flags *pflag.FlagSet) (*EpinioClient,
 	return epinioClient, nil
 }
 
+// ConfigUpdate updates the credentials stored in the config from the
+// currently targeted kube cluster
+func (c *EpinioClient) ConfigUpdate() error {
+	log := c.Log.WithName("ConfigUpdate")
+	log.Info("start")
+	defer log.Info("return")
+	// details := log.V(1) // NOTE: Increment of level, not absolute.
+
+	c.ui.Note().
+		Msg("Updating the stored credentials from the current cluster")
+
+	// 1. API credentials
+	// 2. API certs
+
+	user, password, err := getCredentials(context.Background())
+	if err != nil {
+		c.ui.Exclamation().Msg(err.Error())
+		return nil
+	}
+
+	certs, err := getCerts(context.Background())
```

Comment:
We need to use a single context background I think like in this PR - https://github.com/epinio/epinio/pull/472

Reply:
That is quite a big PR. Where do I find this single context background you want me to use ?
Note also that the command and method in question (`ConfigUpdate`) does not run on the server (API).

It is a pure client command talking directly to the kube cluster.
It has to. As it is retrieving the information needed to talk to the API it cannot use the API by definition.

Given that I am not convinced that I need any special context.

Reply:
@manno :bell: Can you explain things to me a bit more ?

Reply:
as discussed: https://github.com/golang/go/wiki/CodeReviewComments#contexts

Reply:
Found it, `cmd.Context()`.

Reply:
Done, commit c02d7cd89bf7c831cda4fb30beead27d6184c360